casper/src/test/scala/coop/rchain/casper/batch1/StateMergerSpec.scala

```diff
@@ -1,226 +0,0 @@
-package coop.rchain.casper.batch1
-
-import cats.effect.{Concurrent, Sync}
-import cats.implicits._
-import com.google.protobuf.ByteString
-import coop.rchain.casper.helper.TestNode._
-import coop.rchain.casper.helper.TestRhoRuntime.rhoRuntimeEff
-import coop.rchain.casper.merging.{BlockIndex, DagMerger}
-import coop.rchain.casper.syntax._
-import coop.rchain.casper.util.ConstructDeploy
-import coop.rchain.dag.InMemDAG
-import coop.rchain.metrics.{Metrics, NoopSpan, Span}
-import coop.rchain.models.Expr.ExprInstance.GInt
-import coop.rchain.models._
-import coop.rchain.rholang.interpreter.RhoRuntime
-import coop.rchain.rspace.hashing.Blake2b256Hash
-import coop.rchain.shared.Log
-import monix.eval.Task
-import monix.execution.Scheduler.Implicits.global
-import org.scalatest.exceptions.TestFailedException
-import org.scalatest.{FlatSpec, Inspectors, Matchers}
-
-import scala.collection.immutable.ListMap
-
-class StateMergerSpec extends FlatSpec with Matchers with Inspectors with MergeabilityRules {
-
-  it should "respect base mergeability rules" in {
-    baseMergeabilityCases.map(t => evalTestCase(t._1, t._2)).map(_.runSyncUnsafe())
-  }
-
-  it should "respect peak mergeability rules" in {
-    peekMergeabilityCases.map(t => evalTestCase(t._1, t._2)).map(_.runSyncUnsafe())
-  }
-
-  // TODO remove this and enable joinMergeabilityCases
-  it should "declare conflicts when produce touch join" in {
-    touchingJoinShouldBeAlwaysConflicting.map(t => evalTestCase(t._1, t._2)).map(_.runSyncUnsafe())
```

Comment:
So this one is already removed?

Reply:
You can put your little test in https://github.com/rchain/rchain/pull/3422/files#diff-51408dbb416e703418d8fe04e6c3f95d8c8cf2ecd8267e526842def020b355ad

Reply:
@zsluedem I was thinking to add this in deploymerge branch

Reply:
You mean another pr for that?

Reply:
@nzpr Please update PR description with information about what you are thinking to transfer to a new PR.

Reply:
I think given this <https://github.com/rchain/rchain/pull/3424/checks?check_run_id=2743790800> we might need to take a deeper look into joins. I guess makes sense to declare all joins conflicting  as @zsluedem did  and make sure test is passing or to check why they are failing. No need to add my new test, as it looks like it does not cover all cases.